.dependabot/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 1
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "weekly"
+    default_labels:
+      - "dependencies"
+    default_reviewers:
+      - "jimcullenaus"
+      - "lauraducky"
+      - "arthurs"
+      - "evangelinexx"
+      - "drewfreyling"
+      - "gillianmilne"
+      - "chromadream"
+    commit_message:
+      prefix: "fix"
+      prefix_development: "chore"
```

Comment:
what's this?

Reply:
if it's a dev dependency, it will look like `chore(deps-dev): description` otherwise `fix(deps): description`